.github/workflows/on-pr.yml

```diff
@@ -111,7 +111,7 @@ jobs:
     needs:
       - should-run
       - build-test
-    if: needs.should-run.outputs.go == 'true'
+    if: ${{ needs.should-run.outputs.go == 'true' && contains(fromJSON('["release", "master"]'), github.ref_name) }}
```

Comment:
Is it true, that on every RC|beta|release you will have a separate PR to one of these branches?

Reply:
Thanks 👍